armi/reactor/blueprints/blockBlueprint.py

```diff
@@ -119,12 +119,12 @@ def construct(
             if cs["inputHeightsConsideredHot"]:
                 if "group" in c.name:
```

Comment:
I'd like to see a comment on what "group" in c.name means in this context since I haven't seen a use case of this. If this is assigned by the user then this might be very fragile. If it is set in the code then there might an opportunity to make this into a "Flag" or at least make this a constant rather than a string.

Reply:
@jakehader I know we chatted about this offline but I wanted to loop in @ntouran to close the loop. Nick, can you provide context here to resolve Jake's comment?

Reply:
I think that this is out of scope so we will leave as is